tensorflow_addons/activations/gelu.py

```diff
@@ -68,28 +66,9 @@ def gelu(x: TensorLike, approximate: bool = True) -> tf.Tensor:
     """
     warnings.warn(
         "gelu activation has been migrated to core TensorFlow, "
-        "and will be deprecated in Addons 0.13.",
+        "and will be deprecated in Addons 0.13. "
+        "Note that the default of `approximate` changed to `False` in `tf.nn.gelu`.",
         DeprecationWarning,
     )
 
-    x = tf.convert_to_tensor(x)
-
-    if LooseVersion(tf.__version__) >= "2.4":
-        gelu_op = tf.nn.gelu
-        warnings.warn(
-            "Default value of `approximate` is changed from `True` to `False`"
-        )
```

Comment:
I am a bit confused by this warning message, since the default value in TF Addons never changed. Only the default value of `tf.nn.gelu` changed, I updated the warning message to clarify this.